### PR TITLE
Create stub of automated_solution_analysis.md

### PR DESCRIPTION
**Pull request by iHiD**
I'm merging this as I need this file on master for other PRs, but any improvements would be welcome.

### CODE REVIEW

File: pages/automated_solution_analysis.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+### What is it?
+
+We algorithmically analyse solutions to certain exercises to look for optimal solutions, which we can then instantly approve allowing students to move on to the next exercise without delay.
+
+### How does it work?
+
+We use a mixure of techniques to analyze solutions, but most centre around statically analysing the code that students submit. Each language has a different analyzer maintained by a different team, who may approach things in different ways. You may be interested in browsing the code to see how this works:
```
Comment (rpottsoh) [inline pages/automated_solution_analysis.md]: Mixture spelled wrong.  I'm not at a computer or I'd open a PR to fix this.

File: pages/automated_solution_analysis.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+- [C#](https://github.com/exercism/csharp-analyzer)
+- [Erlang](https://github.com/exercism/erlang-analyzer)
+- [Go](https://github.com/exercism/go-analyzer)
+- [JavaScript](https://github.com/exercism/javascript-analyzer)
+- [Java](https://github.com/exercism/java-analyzer)
+- [Perl5](https://github.com/exercism/perl5-analyzer)
+- [Python](https://github.com/exercism/phyton-analyzer)
```
Comment (petertseng) [inline pages/automated_solution_analysis.md]: misspelled (phyton)

Comment (rpottsoh) [reply pages/automated_solution_analysis.md]: #1026 corrects this.

Comment (iHiD) [reply pages/automated_solution_analysis.md]: Thanks @rpottsoh

**Pull request merged**